src/components/Slider/Slider.tsx

```diff
@@ -278,14 +278,15 @@ export const Slider = chakra(
           <ChakraRangeSlider
             // Both slider thumbs need values and should be in an array.
             aria-label={
-              !showLabel
-                ? [`${labelText} - start value`, `${labelText} - end value`]
-                : undefined
-            }
-            // Both slider thumbs need values and should be in an array,
-            // even if it's the same label.
-            aria-labelledby={
-              showLabel ? [`${id}-label`, `${id}-label`] : undefined
+              showLabel
```

Comment:
If the array is defined when both `showLabel`  is true or false, then we can remove the tertiary logic check.

Reply:
Ugh, I meant to include a comment in reference to this array.  When I tried to remove the tertiary logic, the `aria-label` prop complained about the formatting.  I wanted to ask you abnout this.  Perhaps I was just missing some syntax that was throwing it off.

Reply:
I see. Chakra expects it but I think the linter doesn't (rightly) expect an array. Try
```
aria-label={[
              `${labelText} - slider handle for start value`,
              `${labelText} - slider handle for end value`,
            ] as string[]}
```

Reply:
Thanks.  That was perfect.